settings.py

```diff
@@ -1476,6 +1476,11 @@
     "FRI_SMS_SHIFT": "custom.fri.api.shift_custom_content_handler",
     "FRI_SMS_OFF_DAY": "custom.fri.api.off_day_custom_content_handler",
     "UCLA_MESSAGE_BANK": "custom.ucla.api.ucla_message_bank_content",
```

Comment:
i should have said, we can get rid of it, it's only in the test phases right now so it's not actually being used